_strategies/prototyping.md

```diff
@@ -24,37 +24,35 @@ The following principles are meant to guide product teams in the use of technica
 
 ### Avoid Premature Optimization
 
-An acquisition prototype should support a procurement, not become an impediment to it. It should be developed rapidly, using a relatively lightweight approach and at low cost. Guard against over-optimization.
+A prototype should support a procurement and be thought of as a research tool. It should be developed rapidly, using a relatively lightweight approach and at low cost. Guard against over-optimization.
 
-Unlike other kinds of prototypes, acquisition prototypes are not typically used as a way to obtain user feedback, but rather to understand the mechanics behind a hypothetical user's experience. It should approximate a user’s experience to help identify risk, and to help validate technical decisions.
+The government should leverage prototypes to garner obtain user feedback, validate technical decisions, and better understand the organizational barriers to deploying software in modern ways.
 
-If the logic behind modular procurement is to break down large projects into smaller chunks to reduce risk, employ this same approach when prototyping. Instead of thinking of a prototype as an immutable whole, consider how it can be broken into pieces to address different questions or technical risks.
+If the logic behind modular procurement is to break down large projects into smaller chunks to reduce risk, employ this same approach when prototyping. Instead of thinking of a prototype as an immutable whole, consider how it can be broken into pieces to address assumptions about customer value and technical risks. Don't obsess about documentation, but highlight to the  what assumption this prototype was meant to assess, and how this tool measured up against those hypothesis.
```

Comment:
last sentence has some extra words I think. "...highlight which assumption this prototype was meant to assess"

also last word should be "hypotheses"